app/src/main/res/layout/fragment_edit_profile.xml

```diff
@@ -14,11 +14,11 @@
 
         <ImageView
             android:id="@+id/profilePhoto"
-            android:layout_width="@dimen/item_image_view_large"
-            android:layout_height="@dimen/item_image_view_large"
+            android:layout_width="@dimen/avatar_medium"
+            android:layout_height="@dimen/avatar_medium"
```

Comment:
Matching the dimenstions of the avatar icon on both profile fragments for a more consistent look